src/main/java/se/kth/sda/skeleton/user/UserController.java

```diff
@@ -0,0 +1,23 @@
+package se.kth.sda.skeleton.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+
+public class UserController {
+
+    @Autowired
+    UserService userService;
+
```

Comment:
I'm not exactly sure on why we do it, but we had to autowire the service through the constructor like,
`public UserController(@AutoWired UserService **userService**) {
    this.userService = **userService**;
}`

Reply:
Either way should be fine I think. But if you will initialize it this way then with the constructor then, to get the benefit of private access modifier, you should make userService private. Otherwise you might as well not have this constructor around the userService.

Reply:
I will just accept the changes for now and we can make changes when needed tomorrow.